tests/install/test_unattended.py

```diff
@@ -0,0 +1,280 @@
+from datetime import datetime
```

Comment:
This file should be named `test_installation_assistant.py`.

Reply:
Done